src/main/frontend/components/ReviewTile.js

```diff
@@ -0,0 +1,20 @@
+import React from "react"
+
+const ReviewTile = props => {
+  const { title, imgUrl, rating, createdAt, updatedAt } = props.review
```

Comment:
Looks like `updatedAt` can be removed since it isn't being used below.  Maybe in the future we can bring it back in and use it to show the time an `edit` was made 👍

Reply:
Removed updatedAt from state